llvm/lib/Target/AArch64/AArch64Subtarget.h

```diff
@@ -185,6 +185,12 @@ class AArch64Subtarget final : public AArch64GenSubtargetInfo {
            (hasSMEFA64() || (!isStreaming() && !isStreamingCompatible()));
   }
 
+  /// Returns true if the target has access to either the full range of SVE instructions,
+  /// or the streaming-compatible subset of SVE instructions.
+  bool isSVEorStreamingSVEAvailable() const {
+    return hasSVE() || hasSMEFA64() || (hasSME() && isStreaming());
```

Comment:
Is `SMEFA64` relevant?  The feature means SVE and NEON instructions are available when in streaming mode, but you still need to be in streaming mode to execute them. Does it add any new information to the question this function is answering.

Reply:
Good point, I agree that it shouldn't have been included here. I'll remove it.